man/systemd.time.xml

```diff
@@ -122,8 +122,12 @@
     <title>Parsing Timestamps</title>
 
     <para>When parsing, systemd will accept a similar syntax, but expects no timezone specification, unless it is given
-    as the literal string <literal>UTC</literal> (for the UTC timezone) or is specified to be the locally configured
-    timezone. Other timezones than the local and UTC are not supported. The weekday specification is optional, but when
+    as the literal string <literal>UTC</literal> (for the UTC timezone), or is specified to be the locally configured
+    timezone, or the timezone name in the IANA timezone database format. The complete list of timezones
+    supported on your system can be obtained using the <literal>timedatectl list-timezones</literal>
+    (see <citerefentry><refentrytitle>timedatectl</refentrytitle><manvolnum>1</manvolnum></citerefentry>).
+    Using IANA format is recommended over local timezone names, as less prone to errors (eg: with local timezone it's possible to
+    specify daylight savnig time in winter, while it's incorrect). The weekday specification is optional, but when
```

Comment:
savnig => saving

Reply:
thanks. Fixed and squashed all changesets into one.